### PR TITLE
More tests to cover cross-language class lookups.

### PR DESCRIPTION
**Pull request by vext01**
Looks OK?

### CODE REVIEW

File: testing/test_pypy_bridge_scope.py [hunk 1]
```diff
@@ -131,6 +131,79 @@ class C: x = 2
         ''')
         assert php_space.int_w(output[0]) == 2
 
+    def test_php_sees_outer_py_class002(self, php_space):
+        output = self.run('''
+            class C {
+                function __construct() {
+                    $this->x = 3; // should not pick up this class
+                }
```
Comment (ltratt) [inline testing/test_pypy_bridge_scope.py]: I don't understand why the comment is on this line. Isn't the point that this whole class isn't picked up?

File: testing/test_pypy_bridge_scope.py [hunk 2]
```diff
@@ -131,6 +131,79 @@ class C: x = 2
...
+            echo($f()->x);
+        ''')
+        assert php_space.int_w(output[0]) == 2
+
+
+    def test_php_sees_outer_py_class003(self, php_space):
+        output = self.run('''
+            class C {
+                function __construct() {
+                    $this->x = 3; // should not pick up this class
```
Comment (ltratt) [inline testing/test_pypy_bridge_scope.py]: I think this comment should be completely removed?

**Comment by vext01** [comment]
Agreed. Fixed. If OK, will squash.

**Comment by ltratt** [comment]
Squash away!

**Comment by vext01** [comment]
Please merge.

File: testing/test_pypy_bridge_scope.py [hunk 3]
```diff
@@ -131,6 +131,79 @@ class C: x = 2
...
+            echo($f()->x);
+        ''')
+        assert php_space.int_w(output[0]) == 3
+
+
+    def test_php_sees_outer_py_class004(self, php_space):
+        output = self.run('''
+            class C {
+                function __construct($v) {
+                    $this->x = $v; // should not pick up this class
```
Comment (ltratt) [inline testing/test_pypy_bridge_scope.py]: Erp, I think this comment is in the wrong place too.

**Comment by vext01** [comment]
You're right, try now. Squash?

**Comment by ltratt** [comment]
Looks ready for squashing.

**Comment by vext01** [comment]
Squashed, ready for merge.

**Pull request merged**